platform/platform-app/src/main/resources/features.xml

```diff
@@ -28,6 +28,8 @@
           xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
+    <repository>mvn:org.apache.karaf.features/standard/${karaf.version}/xml/features</repository>
+    <repository>mvn:org.ops4j.pax.cdi/pax-cdi-features/${pax.cdi.version}/xml/features</repository>
```

Comment:
Can this repository be removed? I was able to remove it when I was testing [DDF-2132}(https://codice.atlassian.net/browse/DDF-2132), and I didn't see an issues.